README.md

```diff
@@ -14,146 +14,286 @@ https://www.yelp.com/developers/documentation/v3
 npm install yelp-fusion --save
 ```
 
-## Search
+<br/>
+
+## Table of Contents
+Business Endpoint:
+  * [Business Search](#business-search)
+  * [Phone](#phone-search)
+  * [Delivery](#transaction-search)
+  * [Business Details](#business-details)
+  * [Match](#business-match)
+  * [Reviews](#reviews)
+  * [Autocomplete](#autocomplete)
+
+Event Endpoint:
+  * [Event Lookup](#event-lookup)
+  * [Event Search](#event-search)
+  * [Featured Event](#featured-event)
+
+Category Endpoint:
+  * [All Categories](#all-categories)
+  * [Category Details](#category-details)
+
+<br/>
+
+## Business Endpoint 
+
+### Business Search
 ```javascript
 'use strict';
 
 const yelp = require('yelp-fusion');
-
+const apiKey = '<YOUR API KEY>'; 
 const client = yelp.client(apiKey);
 
-client.search({
-  term:'Four Barrel Coffee',
-  location: 'san francisco, ca'
+client.businessSearch({
+  term: 'Four Barrel Coffee',
+  location: 'san francisco, ca',
+  // latitude: 37.7670169511878, 
```

Comment:
I can see why these comments would be convenient. However, I hesitate to include this because I'm certain that I would fail to keep these comments in sync with the main API documentation. When they are out of sync, it could imply that this library doesn't support certain fields when it does. 

The intent of this documentation is to show succinct minimal usage of this library function. Users should refer to the main API documentation for the detailed request models. 

For those reasons, I would like to keep these types of comments out.

Reply:
okay that makes sense!